josemak25/baby-bliss
# Ft get started screens

#### What does this PR do?
add a get started screen
#### Description of Task to be completed?
implement the get started screen
#### How should this be manually tested?
Launch the app and you should see the get started screen immediately after the splash screen

#### Screenshots (if appropriate)
![Simulator Screen Shot - iPhone Xs - 2020-02-23 at 15 56 33](https://user-images.githubusercontent.com/12932317/75114324-6aaaaa80-5655-11ea-9f08-c42b62b59556.png)
